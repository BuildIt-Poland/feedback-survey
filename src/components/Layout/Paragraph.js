import styled from 'styled-components';

import { fontXSmall, fontSmall, fontMedium, spacingSmall, spacingMedium } from '../../styles/designTokens';
import media from '../../styles/media';

const Paragraph = styled.p`
  font-size: ${fontSmall};
  margin: ${spacingSmall} auto;

  footer & {
    font-size: ${fontXSmall};
  }

  ${media.tablet`
    font-size: ${fontMedium};
    margin: ${spacingMedium} auto;

    footer & {
      font-size: ${fontSmall};
    };
  `};

  ${media.laptop`
    footer & {
      font-size: ${fontMedium};
    };
  `};
`;

export default Paragraph;
