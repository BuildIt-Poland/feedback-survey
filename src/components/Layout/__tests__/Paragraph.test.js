import React from 'react';
import { create } from 'react-test-renderer';

import Paragraph from '../Paragraph';

describe('COMPONENT - Layout Paragraph', () => {
  it('renders correctly', () => {
    const component = create(<Paragraph />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
