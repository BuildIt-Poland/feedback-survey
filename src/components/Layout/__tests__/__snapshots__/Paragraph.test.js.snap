// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Layout Paragraph renders correctly 1`] = `
<p
  className="sc-bdVaJa kjwAVT"
/>
`;
